tests/test_qt/test_qkeymap.py

```diff
@@ -28,14 +30,32 @@ def test_qkeysequence2modelkeybinding() -> None:
     app_key = KeyBinding(parts=[KeyMod.Alt | KeyCode.KeyM, KeyCode.KeyK])
     assert qkeysequence2modelkeybinding(seq) == app_key
 
-    seq = QKeySequence(
-        Qt.Modifier.META | Qt.Key.Key_M, Qt.KeyboardModifier.NoModifier | Qt.Key.Key_K
-    )
-    app_key = KeyBinding(parts=[KeyMod.CtrlCmd | KeyCode.KeyM, KeyCode.KeyK])
-    assert qkeysequence2modelkeybinding(seq) == app_key
-
-    seq = QKeySequence(
-        Qt.Modifier.CTRL | Qt.Key.Key_M, Qt.KeyboardModifier.NoModifier | Qt.Key.Key_K
-    )
-    app_key = KeyBinding(parts=[KeyMod.WinCtrl | KeyCode.KeyM, KeyCode.KeyK])
-    assert qkeysequence2modelkeybinding(seq) == app_key
+    with patch.object(_qkeymap, "mac_ctrl_meta_swapped", return_value=True):
```

Comment:
could just directly use `QCoreApplication.setAttribute(Qt.AA_MacDontSwapCtrlAndMeta, False)` to make it clearer what this patch is accomplishing?  (then undo at the end of the test... )

Reply:
ill think about it, might manually set it to a true and then a false as part of the test

Reply:
for some reason doing this messes with the other tests even tho i reset things at the end so im not going thru with it

```
________________________________ test_shortcuts ________________________________

qtbot = <pytestqt.qtbot.QtBot object at 0x7faaea3693d0>
full_app = Application('complete_test_app')

    @pytest.mark.filterwarnings("ignore:QPixmapCache.find:")
    @pytest.mark.skipif(LINUX, reason="Linux keytest not working")
    def test_shortcuts(qtbot: "QtBot", full_app: "FullApp") -> None:
        app = full_app

        win = QMainWindow()
        menu = QModelMenu(app.Menus.EDIT, app=app, title="Edit", parent=win)
        win.menuBar().addMenu(menu)
        qtbot.addWidget(win)
        qtbot.addWidget(menu)

        with qtbot.waitExposed(win):
            win.show()

        copy_action = menu.findAction(app.Commands.COPY)

        with qtbot.waitSignal(copy_action.triggered, timeout=1000):
>           qtbot.keyClicks(win, "C", Qt.KeyboardModifier.ControlModifier)
E           pytestqt.exceptions.TimeoutError: Signal triggered(bool) not emitted after 1000 ms

tests/test_qt/test_qmenu.py:115: TimeoutError
```